applications/zpc/components/zwave_command_classes/src/zwave_command_class_configuration_control.c

```diff
@@ -723,6 +723,16 @@ static sl_status_t
   if (frame_length >= (current_index + 2)) {
     next_id = (configuration_parameter_id_t)((frame[current_index] << 8)
                                              | frame[current_index + 1]);
+    if (next_id == parameter_id) {
```

Comment:
Wouldn't there be the same issue if a device report a next parameter number lower than the current?
```suggestion
    if (next_id <= parameter_id) {
```
And of course also adjust the log message.

Reply:
CC:0070.03.0F.11.019
Next Parameter Number (16 bits)
Configuration parameter identifiers may be assigned in a non-sequential order.